[1;35m [WARNING]: Could not match supplied host pattern, ignoring: oo_masters_to_config[0m
[1;35m[0m
PLAY [Fail openshift_kubelet_name_override for new hosts] **********************
TASK [Gathering Facts] *********************************************************
[0;32mok: [node2.jumpserver.org][0m
[0;32mok: [master.jumpserver.org][0m
[0;32mok: [node1.jumpserver.org][0m
TASK [Fail when openshift_kubelet_name_override is defined] ********************
[1;35m [WARNING]: Failure using method (v2_runner_on_skipped) in callback plugin (<ansible_ui.ansible.callback.PlaybookResultCallBack object at 0x113eec518>): 'skipped'[0m
[1;35m[0m
PLAY [Initialization Checkpoint Start] *****************************************
TASK [Set install initialization 'In Progress'] ********************************
[0;32mok: [node1.jumpserver.org][0m
PLAY [Populate config host groups] *********************************************
TASK [Load group name mapping variables] ***************************************
Task ansible_ui.tasks.execute_playbook[d9b51fe1-ef14-46fe-97c9-67c3a2a311e1] raised unexpected: AttributeError("'NoneType' object has no attribute 'name'",)
Traceback (most recent call last):
  File "/Users/guang/.virtualenvs/ansible/lib/python3.6/site-packages/celery/app/trace.py", line 382, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/Users/guang/.virtualenvs/ansible/lib/python3.6/site-packages/celery/app/trace.py", line 641, in __protected_call__
    return self.run(*args, **kwargs)
  File "/Users/guang/projects/AnsibleUI/ansible_ui/tasks.py", line 18, in execute_playbook
    return playbook.execute(save_history=save_history)
  File "/Users/guang/projects/AnsibleUI/ansible_ui/models/playbook.py", line 185, in execute
    result = runner.run(self.playbook_path)
  File "/Users/guang/projects/AnsibleUI/ansible_ui/ansible/runner.py", line 242, in run
    executor.run()
  File "/Users/guang/.virtualenvs/ansible/lib/python3.6/site-packages/ansible/executor/playbook_executor.py", line 159, in run
    result = self._tqm.run(play=play)
  File "/Users/guang/.virtualenvs/ansible/lib/python3.6/site-packages/ansible/executor/task_queue_manager.py", line 289, in run
    play_return = strategy.run(iterator, play_context)
  File "/Users/guang/.virtualenvs/ansible/lib/python3.6/site-packages/ansible/plugins/strategy/linear.py", line 321, in run
    results += self._wait_on_pending_results(iterator)
  File "/Users/guang/.virtualenvs/ansible/lib/python3.6/site-packages/ansible/plugins/strategy/__init__.py", line 674, in _wait_on_pending_results
    results = self._process_pending_results(iterator)
  File "/Users/guang/.virtualenvs/ansible/lib/python3.6/site-packages/ansible/plugins/strategy/__init__.py", line 117, in inner
    results = func(self, iterator, one_pass=one_pass, max_passes=max_passes)
  File "/Users/guang/.virtualenvs/ansible/lib/python3.6/site-packages/ansible/plugins/strategy/__init__.py", line 433, in _process_pending_results
    queue_cache_entry = (original_host.name, task_result._task)
AttributeError: 'NoneType' object has no attribute 'name'
